oasis-reference/src/test/kotlin/com/empowerops/volition/ref_oasis/OptimizerFixture.kt

```diff
@@ -0,0 +1,37 @@
+package com.empowerops.volition.ref_oasis
+
+import com.empowerops.volition.dto.OASISQueryDTO
+import com.empowerops.volition.dto.RegistrationCommandDTO
+import com.google.common.eventbus.EventBus
+import com.nhaarman.mockitokotlin2.*
+import io.grpc.stub.StreamObserver
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class OptimizerFixture {
+
+
+    //Good reading :
+    // https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying
```

Comment:
find couple good way to use verify with [mockito-kotlin](https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying) and it is a good reference cheat sheet

Reply:
I am not yet fully going through the junit5 vs junit4 yet, which I will do on my own time